src/themes/shared/borders.ts

```diff
@@ -0,0 +1,5 @@
+export const borders = {
+  none: 0,
+  // eslint-disable-next-line @typescript-eslint/naming-convention
+  '1px': '1px solid',
```

Comment:
After checking with Sergiu we can name it `default` because is the one that we will be using.

```suggestion
  'default': '1px solid',
```

Reply:
I just realized that this will make it inconsistent with the chakra theme in the tables for the token preview we are using `<Tr border="1px" borderColor="gray.300">` which has to become `<Tr border="default" borderColor="gray.300">` but it will result either in the chakra version or our version having no borders. Lemme know what you think

Reply:
Something like that would work but it feels a bit off to me
```
const defaultBorder = '1px solid';

export const borders = {
  none: 0,
  default: defaultBorder,
  // eslint-disable-next-line @typescript-eslint/naming-convention
  '1px': defaultBorder,
};

```

Reply:
so then let us call it as you had before `1px`. Thank you!